### PR TITLE
Force inline for ecto schema

### PR DESCRIPTION
**Pull request by Matsa59**
## 📖 Description

Related to #37

## 📋 Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: lib/ex_prosemirror/type.ex [hunk 1]
```diff
@@ -16,6 +16,19 @@ defmodule ExProsemirror.Type do
...
+
+  defp maybe_force_inline(changeset, false), do: changeset
+
+  defp maybe_force_inline(changeset, true) do
+    update_change(changeset, :content, fn content ->
+      if content = List.first(content) do
+        [content]
+      else
+        []
+      end
```
Comment (onixus74) [inline lib/ex_prosemirror/type.ex:30]: could this be one lined ?

**Review by onixus74 — APPROVED** [review]
Looking slick 👍

File: lib/ex_prosemirror/type.ex (hunk 1 above)
Comment (Matsa59) [reply lib/ex_prosemirror/type.ex:30]: I don't think :/ (yeah technically but for readibility)

Comment (Matsa59) [reply lib/ex_prosemirror/type.ex]: I update to

```elixir
case content do
  [content | _] -> content
  _ -> []
end
```

That seem a better solution

**Pull request merged**